.github/workflows/publish.yml

```diff
@@ -184,10 +157,36 @@ jobs:
 
           helm s3 push webapp-*.tgz charts-webapp
 
+      # generates a mapping between branches/tag to wire-build branches
+      - name: Define target branches in wireapp/wire-builds to bump
+        uses: kanga333/variable-mapper@master
+        id: output_target_branches
+        with:
+          key: '${{github.ref}}'
+          # TODO add production and staging once wire-builds has those branches
+          #
+          # "production": {
+          #  "targets": "[\"TDB\"]"
+          # },
+          # "staging": {
+          #   "targets": "[\"TBD\"]"
+          # },
+          map: |
+            {
+              "dev": {
+                "targets": "[\"dev\"]"
+              },
+              "q1-2024": {
+                "targets": "[\"q1-2024\"]"
+              }
+            }
+          export_to: log,output
+
   publish_wire_builds:
     name: Bump webapp chart in wire-builds
     runs-on: ubuntu-latest
     needs: build
+    if: ${{needs.build.outputs.wire_builds_target_branches}}
```

Comment:
This avoids erroring if no target branch is defined (currently erroring when this happens)